### PR TITLE
[APB-1547][LS] Added skeleton client journey flow.

### PR DESCRIPTION
**Pull request by Smudgey**
Updated routes to match new routes in ticket.
Basic skeleton flow, no functionality on the pages.
Added unit test spec for ClientsInvitationController
Renamed InvitationsController to AgentInvitationController
Frontend pages will be cleaned up further by Somme

### CODE REVIEW

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala [hunk 1]
```diff
@@ -34,25 +34,25 @@ import uk.gov.hmrc.play.frontend.controller.FrontendController
 import scala.concurrent.Future
 
 @Singleton
-class InvitationsController @Inject() (
+class AgentInvitationsController @Inject() (
```
Comment (arturopala) [inline app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala]: maybe Agent**s**InvitationControler, matching path?

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala [hunk 2]
```diff
@@ -34,25 +34,25 @@ import uk.gov.hmrc.play.frontend.controller.FrontendController
...
   val messagesApi: play.api.i18n.MessagesApi,
   val authConnector: AuthConnector)(implicit val configuration: Configuration)
   extends FrontendController with I18nSupport with AuthActions {
 
-  import InvitationsController._
+  import AgentInvitationsController._
 
-  val agentsRoot: Action[AnyContent] = Action { implicit request =>
-    Redirect(routes.InvitationsController.enterNino().url)
+  def agentsRoot: Action[AnyContent] = Action { implicit request =>
```
Comment (arturopala) [inline app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala]: I would say `val` is more accurate here then `def`? Any reason to have `def`?

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala (hunk 1 above)
Comment (Smudgey) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala]: For consistency, I should change both, so AgentsInvitationController and ClientsInvitationController, seem okay?

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala (hunk 2 above)
Comment (Smudgey) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/AgentInvitationsController.scala]: I would ask you the same in reverse :) These are controller methods, they execute blocks of code and in some cases they take in arguments too. Should be defs to me

**Pull request merged**